Library/Formula/gpg-agent.rb

```diff
@@ -33,9 +33,46 @@ def install
     system "make", "install"
   end
 
+  def caveats; <<-EOS.undent
+      To replace ssh-agent with gpg-agent, you need to:
+         * ensure that "enable-ssh-support" is in ~/.gnupg/gpg-agent.conf
+         * ensure that "pinentry" in ~/.gnupg/gpg-agent.conf has a graphical PIN entry
+           such as pinentry-mac (needed for GUI programs)
+         * disable the system ssh-agent with
+             launchctl unload -w /System/Library/LaunchAgents/org.openssh.ssh-agent.plist
+         * have launchd start gpg-agent at login
+    EOS
+  end
```

Comment:
It's not loaded and running by default, though; it requires user intervention to do that (which we can detect in the formula using e.g. `launchctl list` and `File.exist?`.